packages/lesswrong/components/editor/Editor.tsx

```diff
@@ -200,6 +196,12 @@ export interface SerializedEditorContents {
   value: any,
 }
 
+export interface FormProps {
+  commentMinimalistStyle?: boolean
+  editorHintText?: string
```

Comment:
Previously this and `hintText` could be JSX elements instead of strings, this worked when we use `renderPlaintextEditor` or `renderDraftJSEditor` but not with CKEditor because it expects the placeholder to be a string (it causes the page to freeze if you pass in a JSX element). I can see why we might want these to not just be plain strings so I'm not that happy about changing this, but in practice all the ones we use can be converted to plain strings so there's not much cost to changing this

cc @b0b3rt